### DISCUSSION TITLE
mutation surport Union Types?

### DISCUSSION THREAD

**Discussion by tangmingkao** [Akryum/vue-apollo#746] [Q&A]
when i use  mutation,can surport Union Types?

just like this in graphiql devtools

`mutation{
  cancleAskOrders: cancelOrder(pairId: 1,direction: ASK){
    status
  }
  cancleBidOrders: cancelOrder(pairId: 1,direction: BID){
    status
  }
}`

**Comment by Akryum** [comment]
You mean if you can execute multiple mutations in one call? Sure!
